scripts/check-config.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var path = require('path');
+var chalk = require('chalk');
+
+var configPath = path.resolve('../config.json');
+
```

Comment:
I understand your concern however npm has already thought through this. `chalk` is added under the `devDependencies` and thusly will only be installed when someone clones down the package and runs npm install in the directory itself, not when a consumer is installing the package.

Also I would make the case that colorizing this specific error message will save you many unnecessary issues in the future b/c people won't read the readme all the way.